src/App.js

```diff
@@ -4,6 +4,27 @@ import questionData from './multipleQuestions.json';
 import logo from './logo.svg';
 import './App.css';
 
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
What does this component do? Does it display the count? Should it manage the state of the count as well?